cmd/main.go

```diff
@@ -94,6 +96,10 @@ func main() {
 		"The minimum duration to wait between two reconciliations for the same node.")
 	flag.DurationVar(&nodeReconciliationRequeueInterval, "node-reconciliation-requeue-interval", 1*time.Minute,
 		"The duration for which nodes are automatically reconciled.")
+	flag.DurationVar(&cacheDefaultTTL, "cache-default-ttl", 15*time.Minute,
+		"The duration for the items in the cache to expire (by default)")
+	flag.DurationVar(&cacheDefaultCleanupInterval, "cache-default-cleanup-interval", time.Minute,
+		"DefaultCleanupInterval triggers cache cleanup (lazy eviction) at this interval.")
```

Comment:
For a user running this application, it's not clear that we're talking about the cache around API calls on the cloud provider.
Also, why are you talking about "default" cache, is there any way to override this default?